docs/API_reference.md

```diff
@@ -580,6 +581,24 @@ Return a POST request JSON response, if one of the requests fails return the err
 If the **value_id** field is not filled the method will return: `{'error': 'missing_parameter'}`.
 In case the language within the `lang` field of the JSON is invalid the method will return `{'error': 'invalid_language'}`.
 
+#### plenty_api_create_property_selection_name <a name=create-property-selection></a>
+
+Create a new name for a property selection value and input the initial value for that language.
+
+[*Required parameter*]:
+
+The **json** parameter contains a single JSON object, containing the selection_Id, language and name.
```

Comment:
When you look at existing methods like `plenty_api_create_attribute_value_name` you can see that I prefer to pass in values as parameters if the number of values is very small. I prefer this over a JSON as building JSONs for 3 values is cumbersome for an application and clutters the code. So, please pass these 3 values in as parameters and build the JSON within the library.

Reply:
We can do that sure